DataFormats/ProtonReco/interface/ProtonTrack.h

```diff
@@ -21,22 +21,33 @@ namespace reco
     public:
       /// Default constructor
       ProtonTrack();
-      /// Constructor from refit parameters, fitted vertex and direction, and longitudinal fractional momentum loss
-      ProtonTrack( double chi2, double ndof, const Point& vtx, const Vector& dir, float xi, const CovarianceMatrix& cov = CovarianceMatrix() );
+
+      /// Constructor from refit parameters, fitted vertex and momentum, and longitudinal fractional momentum loss
+      ProtonTrack( double chi2, double ndof, const Point& vtx, const Vector& momentum, float xi, const CovarianceMatrix& cov = CovarianceMatrix() );
 
       /// Indices to the covariance matrix
+      // TODO: what is num_indeces, where used ??
```

Comment:
Old C trick if you want to avoid redefining a variable (outside to the `enum`), handling the number of elements within the `enum`.